.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "seastar"]
 	path = seastar
-	url = ../seastar
+	url = https://github.com/michoecho/seastar
 	ignore = dirty
```

Comment:
Please post that on seastar.git so other seastar users can review.

Reply:
This was done and the seastar change was merged.